app/views/devise/passwords/edit.html.slim

```diff
@@ -14,6 +14,9 @@
         = f.input :reset_password_token, as: :hidden
         = f.full_error :reset_password_token
         = f.input :password, label: 'New password', required: true, autofocus: true
+        = render 'devise/shared/password_strength'
         = f.input :password_confirmation, label: 'Confirm your new password', required: true
         = f.button :submit, 'Change my password'
       = render 'devise/shared/links'
+
+== javascript_include_tag 'misc/pw-strength'
```

Comment:
That seems quite suboptimal if I understand this approach. The thing is, we don't want the JavaScript to be on the page at all if not needed, not just not invoked